rsrcdump/packutils.py

```diff
@@ -4,20 +4,22 @@
 import struct
 
 class Unpacker:
+    __slots__ = ("data", "offset")
+
     def __init__(self, data: bytes, offset: int=0) -> None:
         self.data = data
         self.offset = offset
 
-    def unpack(self, fmt: str) -> tuple:
+    def unpack(self, fmt: str) -> tuple[int, ...]:
```

Comment:
We can't know, statically, if the first item in the tuple is going to be an int. The exact types depend on what `struct.unpack_from` will infer from `fmt`. So I suggest `tuple[Any, ...]`.

Reply:
Fixed in e907f63